datalad/support/annexrepo.py

```diff
@@ -12,30 +12,31 @@
 
 """
 
-from os import linesep
-from os.path import join as opj, exists, relpath, islink, realpath, lexists
-import logging
 import json
-import re
+import logging
 import os
+import re
 import shlex
+from os import linesep
+from os.path import join as opj, exists, islink, realpath, lexists
 from subprocess import Popen, PIPE
+
 #import pexpect
 
 from functools import wraps
 
 from six import string_types
 from six.moves import filter
 from six.moves.configparser import NoOptionError
-from six.moves.urllib.parse import quote as urlquote
 
 from ..dochelpers import exc_str
 from ..utils import auto_repr
-from .gitrepo import GitRepo, normalize_path, normalize_paths, GitCommandError
+from datalad.support.gitrepo import GitRepo, normalize_path, normalize_paths, GitCommandError
```

Comment:
would be nice to observe at least local consistency - i.e. using relative paths where makes sense.  Here in particular -- I don't think that annexrepo.py would any time soon migrate to another from gitrepo.py directory. Moreover above we have relative import as well.  Best to stay consistent at least